src/bindings.rs

```diff
@@ -413,6 +484,41 @@ fn deserialize_value(
                 deserialize_object(context, r)
             }
         }
+        // BigInt
+        #[cfg(feature = "num-bigint")]
+        TAG_BIG_INT => {
+            let mut int: i64 = 0;
+            let ret = unsafe { q::JS_ToBigInt64(context, &mut int, *r) };
+            if ret == 0 {
+                Ok(JsValue::BigInt(BigInt {
+                    inner: BigIntOrI64::Int(int),
+                }))
+            } else {
+                if ret == -1 {
+                    return Err(ValueError::Internal("Internal BigInt error".into()));
```

Comment:
```suggestion
                    return Err(ValueError::Internal("BigInt conversion error".into()));
```

Edit: actually, should this if still be there?

Reply:
Yeah, I think this check is not needed at this point